interpret_date.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'activerecord', ['>= 5.1', '< 7']
-  spec.add_development_dependency "bundler", "~> 1.6"
-  spec.add_development_dependency "rake", "~> 12"
+  spec.add_runtime_dependency 'activerecord', '>= 7'
```

Comment:
We should leave in here that it still supports old versions of Rails. That doesn't hurt anything. You can actually just making this `<= 7`

Reply:
Amended in c944626.